README.md

```diff
@@ -56,21 +56,21 @@ CI runs a series of steps;  this the sequence to do it locally, along with some
 1. **Pull down the latest docker containers**
 
     ```
-    docker-compose pull
+    docker compose pull
```

Comment:
We've been steadily updating documentation to reference the modern incantation for Docker Compose, so I did the same here.